Sources/Swift/Core/Tools/ViewCapture/SentryUIRedactBuilder.swift

```diff
@@ -9,8 +9,23 @@ import WebKit
 #endif
 
 final class SentryUIRedactBuilder {
+    // MARK: - Constants
+
+    /// Class object identifier for ``CameraUI.ChromeSwiftUIView``, if it exists.
+    ///
+    /// This object identifier is used to identify views of this class type during the redaction process.
+    /// This workaround is specifically for Xcode 16 building for iOS 26 where accessing CameraUI.ModeLoupeLayer
+    /// causes a crash due to unimplemented init(layer:) initializer.
+    private static let cameraSwiftUIViewClassObjectId: ObjectIdentifier? = {
+        guard let classType = NSClassFromString("CameraUI.ChromeSwiftUIView") else {
```

Comment:
`h`: When looking at the crash in https://github.com/getsentry/sentry-cocoa/issues/5647 it could be that simply calling `NSClassFromString` could still lead to crashes. We had a weird issue in the past when swizzling UIViewControllers (https://github.com/getsentry/sentry-cocoa/issues/3798) and therefore we don't call it in the SentrySubClassFinder for excluded classes https://github.com/getsentry/sentry-cocoa/blob/15a6325c9249637592cd76d09c4c1cc45489b504/Sources/Sentry/SentrySubClassFinder.m#L72-L77

I even opened a bug in the Swift repo but it never got fixed: https://github.com/swiftlang/swift/issues/72657. I'm unsure if it's worth the effort to do this right now. Simply doing a check if we're running in iOS 26 could be safer.